docs/input_file_formats.md

```diff
@@ -4,60 +4,46 @@
 To run VaSeBuilder, specific VaSeBuilder input files might need to be provided depending on the selected run mode.<br />
 <br />
 
-
 ### Donor variant and donor alignment list files
 The donor variant list file should contain a list of paths to a variant, VCF or BCF, file with one path per file line.
 The donor alignment list file should be similar and contain one path to a donor alignment file per file line.
 The files listed will be used as donor variant and alignments files respectively when constructing variant contexts.
 Only paths to the variant/alignment files should be provided, not the index files. These are assumed to be in the same 
 directory with the same name, i.e.: sample.bam & sample.bam.bai
-<br /><br />
 
 ### Genome reference
 As VaSeBuilder can work with both BAM and CRAM alignment files, users need to supply a genome reference in fasta format for each VaSeBuilder run. This genome reference should be the same as the reference used during the mapping process to create the acceptor and donor alignment files.
 
-
 ### Donor FastQ list file
-A donor fastq list file needs to be provided when running VaSeBuilder AC-mode. Currently (September 23rd, 2019) this 
-file requires two columns without a header. The first column should contain the paths to R1 files, the second column
- the paths to R2 files:
+_Note: Using VaSeBuilder-produced FastQ spike-in building blocks is deprecated in favor of using BAMs, as this allows for more flexibility and provides more information._
 
-```
-/path/to/sample1_R1.fq &emsp;/path/to/sample1_R2.fq
-/path/to/sample2_R1.fq &emsp;/path/to/sample2_R2.fq
-/path/to/sample3_R1.fq &emsp;/path/to/sample3_R2.fq
-```
-
-Each row in this file therefore represents a single sample.
```

Comment:
Removed this line just because it's a little ambiguous, since samples can be represented by multiple lines if they're split across lanes and such.